style2.css

```diff
@@ -0,0 +1,364 @@
+  margin-top: 5rem;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.contact .contact-header p {
+  font-size: 1.5rem;
+  margin: 0;
+}
+
+.contact .socialmedia-icons h5 {
+  margin: 0;
+}
+
+.contact .socialmedia-icons a {
+  text-decoration: none;
+}
+
+.contact .contact-content {
+  height: 50vh;
+  margin: 0rem 2rem;
+  text-align: center;
+}
+
+.contact .contact-content p {
+  font-size: 1.3rem;
+}
+
+.contact .form-container {
+  margin: 0 auto;
+  width: 100%;
+  clear: both;
+}
+
+.contact .form-container form label {
+  float: left;
+}
+
+.contact .form-container form input {
+  border: #000 solid 0.0625rem;
+  border-radius: 0.5rem;
+  height: 1.5rem;
+  margin: 0 0 0.8rem 0;
+  width: 100%;
+  clear: both;
+}
+
+.contact .form-container form textarea {
+  border: #000 solid 0.0625rem;
+  border-radius: 0.5rem;
+  margin-bottom: 0.8rem;
+  width: 100%;
+  clear: both;
+}
+
+footer {
+  margin-top: 10rem;
+  height: 4.8vh;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: space-evenly;
+      -ms-flex-pack: space-evenly;
+          justify-content: space-evenly;
+  color: white;
+  background-color: #000;
+}
+
+footer .footer-copyright {
+  padding-top: 0.5rem;
+  text-align: center;
+}
+
+footer .socialmedia-icons i {
+  color: #fff;
+}
+
+.socialmedia-icons a {
+  font-size: 1rem;
+  color: #000;
+}
+
+button {
+  border: #000 solid 0.0625rem;
+  background-color: #fff;
+  padding: 1rem;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 0.85rem;
+}
+
+.anchor-btn {
+  border: 0.0625rem solid #000;
+  background-color: #fff;
+  padding: 1rem;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 0.85rem;
+  color: black;
+}
+
+/* Media queries */
+/* Desktop */
+@media only screen and (min-width: 992px) {
+  nav p {
+    font-size: 1.7rem;
+  }
+  nav ul li {
+    font-size: 1rem;
+  }
+  header .header-background {
+    background-image: url(./images/background.jpg);
+    background-size: cover;
+  }
+  header .header-text {
+    font-size: 3rem;
+  }
+  .about {
+    height: 70vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: row;
+            flex-direction: row;
+    -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+            justify-content: space-between;
+    gap: 0 2rem;
+    margin: 5rem 10rem 0 10rem;
+  }
+  .about .about-left {
+    width: 100%;
+    height: 80%;
+  }
+  .about .about-right {
+    padding-left: 2rem;
+    line-height: 2rem;
+  }
+  .proficiency {
+    margin: 0rem 10rem 0 10rem;
+  }
+  .contact {
+    height: 50vh;
+    margin: 5rem 10rem 0 10rem;
+  }
+  .contact .contact-header {
+    margin-bottom: 1rem;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
```

Comment:
Your indentation is a little out of line here. Please fix this.

Reply:
done